ledger-service/http-json/src/main/scala/com/digitalasset/http/WebSocketService.scala

```diff
@@ -640,7 +711,9 @@ object WebSocketService {
   private[WebSocketService] final class InitialEnrichedContractKeyWithStreamQuery private ()(
       implicit ec: ExecutionContext
   ) extends EnrichedContractKeyWithStreamQuery[Unit] {
-    override def removePhantomArchives(request: NonEmptyList[CKR[LfV]]) = Some(Set.empty)
+    override def removePhantomArchives(request: ResolvedContractKeyStreamRequest[Unit, LfV]) = Some(
+      Set.empty
+    )
```

Comment:
Nitpick, sometimes scalafmt does silly things.

```suggestion
    override def removePhantomArchives(request: ResolvedContractKeyStreamRequest[Unit, LfV]) =
      Some(Set.empty)
```